test/test.render.js

```diff
@@ -55,3 +55,39 @@ tap.test('renderPage', (t) => {
     }
   }, t.end);
 });
+
+tap.test('renderCollection', (t) => {
+  const pr = new PagedataRenderer('apiKey', { path: __dirname, host: 'http://localhost:8081' });
+  async.autoInject({
+    server(done) {
+      const server = new Hapi.Server();
+      server.connection({
+        host: 'localhost',
+        port: 8081
+      });
+      server.route({
+        path: '/api/pages',
+        method: 'get',
+        handler(request, reply) {
+          t.equal(request.query.parentPageSlug, 'collection-slug');
+          return reply(null, [
+            { slug: 'page1-slug', text: 'Hello World 1' },
```

Comment:
this isn't very realistic.  it'll have something like: `content: { title: 'blah', name: 'bob' }`